zxlive/settings_dialog.py

```diff
@@ -71,13 +71,19 @@ class SettingsData(TypedDict):
     'sqasm-no-simplification': "Spider QASM (no simplification)",
 }
 
+combo_true_false = {
+    'True': 'True',
+    'False': 'False'
+}
+
 
 general_settings: list[SettingsData] = [
     {"id": "path/custom-rules", "label": "Custom rules path", "type": FormInputType.Folder},
     {"id": "color-scheme", "label": "Color scheme", "type": FormInputType.Combo, "data": color_scheme_data},
     {"id": "tab-bar-location", "label": "Tab bar location", "type": FormInputType.Combo, "data": tab_positioning_data},
     {"id": "snap-granularity", "label": "Snap-to-grid granularity", "type": FormInputType.Combo, "data": snap_to_grid_data},
     {"id": "input-circuit-format", "label": "Input Circuit as", "type": FormInputType.Combo, "data": input_circuit_formats},
+    {"id": "previews-show", "label": "Show rewrite previews","type": FormInputType.Combo, "data": combo_true_false}
```

Comment:
this will look better as a checkbox instead of a dropdown with true or false

Reply:
Or potentially Toggle switch, but I think it's a great issue for someone during the hackathon: #321